pkg/controllers/user/logging/clusterlogging.go

```diff
@@ -146,3 +142,25 @@ func (c *ClusterLoggingSyncer) updateEmbeddedEndpoint() {
 		}
 	}(ctx, cancel)
 }
+
+func cleanResource(ds v1beta2.DaemonSetInterface, sa v1.ServiceAccountInterface, cm v1.ConfigMapInterface, crb rbacv1.ClusterRoleBindingInterface, cl v3.ClusterLoggingLister, pl v3.ProjectLoggingLister) error {
+	allDisabled, err := utils.IsAllLoggingDisable(cl, pl)
+	if err != nil {
+		return err
+	}
+
+	if allDisabled {
```

Comment:
we can clean all resource by simply remove the cattle-logging namespace, what do you think?

Reply:
Nice advise, I will update.